pkgs/misc/drivers/openzwave/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub
+, doxygen, libxml2, git, which, pkgconfig, graphviz-nox
+, udev }:
+
+let
+  version = "2018-04-04";
+
+in stdenv.mkDerivation rec {
+  name = "openzwave-${version}";
+
+  src = fetchFromGitHub {
+    owner = "OpenZWave";
+    repo = "open-zwave";
+    rev = "ab5fe966fee882bb9e8d78a91db892a60a1863d9";
+    sha256 = "0yby8ygzjn5zp5vhysxaadbzysqanwd2zakz379299qs454pr2h9";
+  };
+
+  nativeBuildInputs = [ doxygen libxml2 git which pkgconfig graphviz-nox ];
+  buildInputs = [ udev ];
+
+  preBuild = ''
+    makeFlagsArray+=("DESTDIR=$out")
+    makeFlagsArray+=("PREFIX=$out")
+    makeFlagsArray+=("pkgconfigdir=lib/pkgconfig")
+    makeFlagsArray+=("sysconfigdir=etc/openzwave")
+  '';
+
+  postPatch = ''
+    sed -i "s#/etc/openzwave/#$out/etc/openzwave/#" cpp/src/Options.cpp
+  '';
+
+  fixupPhase = ''
+    mv $out/$out/bin $out/bin
+    mv $out/$out/etc $out/etc
+    mv $out/$out/lib/libopenzwave.so* $out/lib/
+    mv $out/$out/include $out/include
+    mv $out/$out/share $out/share
+
+    rm -r $out/nix
+
+    sed -i "s#pcfile=#pcfile=$out/#" $out/bin/ozw_config
+  '';
+
+  meta = with stdenv.lib; {
+    description = "C++ library to control Z-Wave Networks via a USB Z-Wave Controller";
+    homepage = https://github.com/OpenZWave/open-zwave;
```

Comment:
Maybe http://www.openzwave.net/ instead?

Reply:
I've honestly missed that that one exists because it's not coming up top when I've been doing searches :D